apps/chat/endpoints/v1/chat.py

```diff
@@ -0,0 +1,40 @@
+'''
+Path functions for /apps/user/v1/user
+'''
+from typing import List
+
+from fastapi import status, APIRouter, Depends
+from fastapi.responses import JSONResponse
+from redis.asyncio import Redis
+
+from ...models.chat import Chat, DBChat
+from ...models.message import Message
+from services.redis_cache import get_messages, get_redis
+
+router = APIRouter()
+
+
+@router.get(
+    '/',
+    response_model=List[DBChat],
```

Comment:
Please insert `dependencies=[auth_user]` for security. It automatically throws exception when unauthorized user accessed this API.